test/test_afk.rb

```diff
@@ -34,10 +54,13 @@ def test_departs_if_dead
   end
 
   def test_exits_if_low_health
+    setup_settings({})
     expected_messages = ['health', 'avoid all', 'exit']
-    self.health = 20
 
     run_script('afk')
+    sleep 0.4
```

Comment:
This sleep is causing an issue with threading. Im going to update this test so that it is skipped instead. `afk` pauses during start up until health and spirit are above a threshold. For this test to work, there needs to be a way to modify the health while `afk` is running.